android/samples/hello-triangle/app/build.gradle

```diff
@@ -1,9 +1,69 @@
-apply plugin: 'com.android.application'
+// This script accepts the following parameters:
+//
+// filament_tools_dir
+//     Path to the Filament distribution/install directory for desktop
```

Comment:
You could add a similar check to what Tungsten is doing to check for `matc` binary [here](https://github.com/google/filament/blob/master/tools/tungsten/plugin/build.gradle#L38-L41)

Reply:
Done!